frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -238,6 +265,43 @@
             }
 
             function createComponentsTable() {
+                var columns = [
+                    {"title": "Vendor", data: "vndrs"},
+                    {"title": "Component Name", data: "name", render: {display: renderComponentNameLink}},
+                    {"title": "Main Licenses", data: "lics", render: {display: renderLicenseLink}},
+                    {"title": "Component Type", data: "cType"},
+                    {"title": "Actions", data: "id", render: {display: renderComponentActions}}
+                ];
+                var order = [[1, 'asc']];
+                var columnDefs =  [];
+
+                if (window.codescoopEnabled) {
```

Comment:
Why is't this handled in jsp?

Reply:
This looks like hard to maintain code. E.g. hard to test whether the column widths and styles are still good if no CS support is present.

Reply:
@maxhbr regarding @mcjaeger request to deal with table UI we need to catch it here to make table init process match to column size

its totally easy to see same result as before if codescoop is not enabled: page will be same as now on your instances

Reply:
I think the issue is that if someone now changes the table, without having access to codescoop, there is no chance of testing if the changes break the codescoop view.

Reply:
updated

Reply:
@mcjaeger how we should proceed here?

Reply:
@maxhbr what would be alternatives here? I think some logic is necessary in JavaScript as the altering of the table.

Reply:
One could move the logic to the jsp level, e.g. by `<core_rt:if test="${not empty CODESCOOP_URL && not empty CODESCOOP_TOKEN}">`.